From f08383d1da55a3e0efef2d372d8cf124cc98a860 Mon Sep 17 00:00:00 2001
From: Jordan Crouse <jcrouse@codeaurora.org>
Date: Tue, 27 Dec 2011 11:01:55 +0100
Subject: [PATCH] msm: kgsl: Replace fget_light/fput_light with fget/fput

The use case really doesn't fit the critera for fget_light and
fput_light, and those symbols are not declared as external symbols
anyway, so it seems to be more correct to use the heavier functions.

Change-Id: I27db52053e071060bb8e0c17d2d347ac295a5abc
Signed-off-by: Jordan Crouse <jcrouse@codeaurora.org>
---
 drivers/video/msm/gpu/kgsl_adreno205_hc/kgsl.c |    5 ++---
 1 files changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/video/msm/gpu/kgsl_adreno205_hc/kgsl.c b/drivers/video/msm/gpu/kgsl_adreno205_hc/kgsl.c
index 1bf845c..9dfb4e5 100644
--- a/drivers/video/msm/gpu/kgsl_adreno205_hc/kgsl.c
+++ b/drivers/video/msm/gpu/kgsl_adreno205_hc/kgsl.c
@@ -1371,7 +1371,6 @@ static int kgsl_get_phys_file(int fd, unsigned long *start, unsigned long *len,
 			      unsigned long *vstart, struct file **filep)
 {
 	struct file *fbfile;
-	int put_needed;
 	int ret = 0;
 	dev_t rdev;
 	struct fb_info *info;
@@ -1380,7 +1379,7 @@ static int kgsl_get_phys_file(int fd, unsigned long *start, unsigned long *len,
 	if (!get_pmem_file(fd, start, vstart, len, filep))
 		return 0;
 
-	fbfile = fget_light(fd, &put_needed);
+	fget(fd);
 	if (fbfile == NULL) {
 		KGSL_CORE_ERR("fget_light failed\n");
 		return -1;
@@ -1399,7 +1398,7 @@ static int kgsl_get_phys_file(int fd, unsigned long *start, unsigned long *len,
 		ret = -1;
 	}
 
-	fput_light(fbfile, put_needed);
+	fput(fbfile);
 
 	return ret;
 }
-- 
1.7.5.4

